plugins/experimental/stale_while_revalidate/stale_while_revalidate.c

```diff
@@ -56,85 +60,41 @@ typedef struct {
 } config_t;
 
 typedef struct {
-  time_t date, stale_while_revalidate, stale_on_error, max_age;
+  time_t date, stale_while_revalidate, stale_on_error, max_age, rmt_date;
```

Comment:
What does `rmt` stand for?

Reply:
Remote.

Reply:
So you are changing `date` to mean the cache date and `rmt_date` is the header sent by the origin?

Reply:
Yep.   Age is the date-from-last-retrieve and rmt_date is the Date: header from the far side (used also for the far side to determine If-Modified-Since).

Having one date that somehow matches two separate distributed sites is always a bug waiting to happen.  Ryan has seen that go off before.   Maybe even with this feature?